kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -1,39 +1,77 @@
 <template>
 
-  <li>
-    <a
-      :href="link"
-      class="core-menu-option"
-      role="menuitem"
-      :class="$computedClass(optionStyle)"
-      :tabindex="link ? false : '0'"
-      @click="conditionalEmit"
-      @keydown.enter="conditionalEmit"
-    >
-      <slot>
-        <KLabeledIcon>
-          <template v-if="icon" #icon>
-            <KIcon
-              :icon="icon"
-              :color="optionIconColor"
-            />
-          </template>
-          <div v-if="label">{{ label }}</div>
-        </KLabeledIcon>
-        <div
-          v-if="secondaryText"
-        >{{ secondaryText }}</div>
-      </slot>
-    </a>
-  </li>
+  <div>
+    <!-- routes that have subpaths -->
+    <li v-if="subRoutes && subRoutes.length > 0">
+      <a
```

Comment:
Is this an `a` tag just to make it appear clickable?

Reply:
I thought it should actually be an <a> tag because it has to do with page navigation, like from an accessibility perspective, but maybe it doesn't have to be...

Reply:
Yeah my only thought is that semantically it's not a link, so having it as an `a` feels a bit odd. But it should definitely be _something_ that has a pointer property that makes it look clickable, and will be in the tab sequence for keyboard navigation.